frontend/public/actions/dashboards.ts

```diff
@@ -71,15 +73,27 @@ export const watchURL: WatchURLAction = (url, fetch = coFetchJSON) => (dispatch,
   }
 };
 
+export const watchAlerts: WatchAlertsAction = () => (dispatch, getState) => {
+  const isActive = isWatchActive(getState().dashboards, RESULTS_TYPE.ALERTS, ALERTS_KEY);
+  dispatch(activateWatch(RESULTS_TYPE.ALERTS, ALERTS_KEY));
+  if (!isActive) {
+    const alertMgrURL = `${window.SERVER_FLAGS.alertManagerBaseURL}/api/v2/alerts?silenced=false&inhibited=false`;
```

Comment:
we dont even need to attempt fetching alerts if `window.SERVER_FLAGS.alertManagerBaseURL` is not defined. Take a look at `watchPrometheusQuery`

Reply:
Done!